histomicstk/filters/shape/clog.py

```diff
@@ -2,7 +2,7 @@
 import scipy as sp
 
 
-def cLoG(I, Mask, SigmaMin=30*1.414, SigmaMax=50*1.414):
+def clog(I, Mask, SigmaMin=30*1.414, SigmaMax=50*1.414):
```

Comment:
Rename `I` to `im_input`, `Mask` to `mask`, `SigmaMin` to `sigma_min`, `SigmaMax` to `sigma_max`.

Follow the same for other functions. 

Using PyCharm's refactor functionality will make things easier